service/kvmconfig/v3/resources.go

```diff
@@ -1,23 +1,25 @@
-package kvmconfigv3
+package v3
 
 import (
 	"github.com/cenkalti/backoff"
 	"github.com/giantswarm/certs"
-	"github.com/giantswarm/kvm-operator/service/cloudconfigv3"
-	"github.com/giantswarm/kvm-operator/service/resource/configmapv3"
-	"github.com/giantswarm/kvm-operator/service/resource/deploymentv3"
-	"github.com/giantswarm/kvm-operator/service/resource/ingressv2"
-	"github.com/giantswarm/kvm-operator/service/resource/namespacev2"
-	"github.com/giantswarm/kvm-operator/service/resource/pvcv2"
-	"github.com/giantswarm/kvm-operator/service/resource/serviceaccountv3"
-	"github.com/giantswarm/kvm-operator/service/resource/servicev2"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
 	"github.com/giantswarm/operatorkit/framework/resource/metricsresource"
 	"github.com/giantswarm/operatorkit/framework/resource/retryresource"
 	"github.com/giantswarm/randomkeys"
 	"k8s.io/client-go/kubernetes"
+
+	// TODO copy missing v2 resources and veresion messagecontext.
```

Comment:
As todo says. To be done in follow up PR. I don't want to copy here and introduce more clutter. I didn't version messagecontext now, because that may break flow between v2 and v3 resources.